kubernetes/samples/charts/apache-samples/README.md

```diff
@@ -0,0 +1,8 @@
+# Apache load balancer samples
+
+The sample package contains two samples that uses the [Apache helm chart](../apache-webtier/README.md). The samples uses the Docker image for the Apache HTTP Server with the 12.2.1.3.0 Oracle WebLogic Server Proxy Plugin.  See the details in [Apache HTTP Server with Oracle WebLogic Server Proxy Plugin on Docker](https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-webtier-apache).
```

Comment:
The sample package contains two samples that uses -> The sample package contains two samples that use